clients/admin-ui/src/features/config-wizard/ConfigWizardWalkthrough.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Button, Divider, Stack } from "@fidesui/react";
+import { useRouter } from "next/router";
+import React, { useState } from "react";
+
+import { CloseSolidIcon } from "~/features/common/Icon";
+
+import Stepper from "../common/Stepper";
+import steps from "./constants";
+import OrganizationInfoForm from "./OrganizationInfoForm";
+
+const ConfigWizardWalkthrough = () => {
+  const router = useRouter();
+  const [step, setStep] = useState<number>(1);
+
+  const handleCancelSetup = () => {
+    // Save progress
+    router.push("/");
+  };
+
+  const handleChangeStep = (formStep: number) => {
+    // Save info between steps for submission to API with all info
+    // or are they different api calls at each step?
```

Comment:
hmm good question, I'm not sure either!

Reply:
Also going to address this in a separate PR ^ Probably the same as the cancelling.